File: LeetCode/Python/0561_Array_Partition_I.py
class Solution:
    def arrayPairSum(self, nums: List[int]) -> int:
        if len(nums) == 2:
            return min(nums)
        
        _sum = 0
        nums.sort()
        
        while nums:
            nums.pop()
            _sum += nums.pop()
            
        return _sum

File: LeetCode/Java/0561_Array_Partition_I.java
class Solution {
    public int arrayPairSum(int[] nums) {
        if (nums.length == 2)
            return Math.min(nums[0], nums[1]);
        
        Arrays.sort(nums);
        int sum = 0;
        
        for (int i = 0; i < nums.length; i += 2)
            sum += nums[i];
        
        return sum;
    }
}
